hudi-client/src/main/java/org/apache/hudi/table/action/rollback/ListingBasedRollbackHelper.java

```diff
@@ -182,12 +162,12 @@ private HoodieRollbackStat mergeRollbackStat(HoodieRollbackStat stat1, HoodieRol
    * Common method used for cleaning out parquet files under a partition path during rollback of a set of commits.
    */
   private Map<FileStatus, Boolean> deleteCleanedFiles(HoodieTableMetaClient metaClient, HoodieWriteConfig config,
-      Map<FileStatus, Boolean> results, String commit, String partitionPath) throws IOException {
+                                                      String commit, String partitionPath) throws IOException {
+    final Map<FileStatus, Boolean> results = new HashMap<>();
     LOG.info("Cleaning path " + partitionPath);
     FileSystem fs = metaClient.getFs();
-    String basefileExtension = metaClient.getTableConfig().getBaseFileFormat().getFileExtension();
     PathFilter filter = (path) -> {
-      if (path.toString().contains(basefileExtension)) {
+      if (path.toString().contains(HoodieFileFormat.PARQUET.getFileExtension())) {
```

Comment:
@lw309637554 any reason for changing this to PARQUET? (or was that me?)

Reply:
Get it. it is because master merge  do not handle well. Let me fix it

Reply:
done